frontend/webpack.config.ts

```diff
@@ -2,17 +2,18 @@
 import * as webpack from 'webpack';
 import * as path from 'path';
 import * as glob from 'glob';
+import { default as chalk } from 'chalk';
```

Comment:
Cool, but do we really _need_ it?

Reply:
Note that this change comes from unmerged PR #1528.

It's only needed to visually highlight individual plugins that are part of the webpack build.

![log-active-plugins](https://user-images.githubusercontent.com/648971/57157311-a6336a00-6de0-11e9-898e-33751efaba28.png)

If you think we don't need that, I can remove it from #1528. However, the project already uses `chalk` as a transitive dependency, check out the output of `yarn why chalk`.

Reply:
If it's already a transitive dep, then it's fine I suppose.